cdc/processor/processor.go

```diff
@@ -475,6 +475,7 @@ func isProcessorIgnorableError(err error) bool {
 // the `state` parameter is sent by the etcd worker, the `state` must be a snapshot of KVs in etcd
 // The main logic of processor is in this function, including the calculation of many kinds of ts, maintain table pipeline, error handling, etc.
 func (p *processor) Tick(ctx cdcContext.Context, state *orchestrator.ChangefeedReactorState) (orchestrator.ReactorState, error) {
+	p.changefeed = state
```

Comment:
Why moving `p.changefeed = state` from the original place to this line.

Reply:
initialize the `changefeed` pointer as soon as possible, to prevent call on nil `changefeed`.

Reply:
Actually, we should refactor the `newProcessor` method to attach the `changefeed` immediately, to prevent the nil pointer, I will do it in the later days.